File: ios/Cpp_to_RN/SendCpp_to_RN.m
#import <React/RCTBridgeModule.h>
#import <Foundation/Foundation.h>
#import "UIKit/UIKit.h"
@interface RCT_EXTERN_MODULE(SendCpp_to_RN, NSObject)

RCT_EXTERN_METHOD(fromCpp:(RCTResponseSenderBlock)successCallback)

@end

File: ios/Cpp_to_RN/SendCpp_to_RN.swift
import Foundation

@objc(SendCpp_to_RN)
class SendCpp_to_RN : NSObject {
    
  @objc static func requiresMainQueueSetup() -> Bool {
        return false
    }
  
  @objc func fromCpp(_ successCallback: RCTResponseSenderBlock) -> Void {
    successCallback([NSNull(), WrapCpp_to_RN().wrapSayHello() as Any])
    }

}
